build.assets/Dockerfile

```diff
@@ -190,7 +190,7 @@ RUN (curl -L https://github.com/bats-core/bats-core/archive/v1.2.1.tar.gz | tar
 
 # Install shellcheck.
 RUN scversion='v0.8.0' && \
-    curl -sSL "https://github.com/koalaman/shellcheck/releases/download/$scversion/shellcheck-$scversion.linux.x86_64.tar.xz" | \
+    curl -sSL "https://github.com/koalaman/shellcheck/releases/download/$scversion/shellcheck-$scversion.linux.$(if [ "$BUILDARCH" = "amd64" ]; then echo "x86_64"; else echo "aarch64"; fi).tar.xz" | \
```

Comment:
since each arch is built separately, can't we just use `uname -m` ?

Reply:
We could, but each platform returns different names, and the packages have different names too. We already have one mapping here https://github.com/gravitational/teleport/blob/master/build.assets/Makefile#L36-L41
I know this line works as I copied it from below, but even then I had to change `aarch_64` to `aarch64` as the name of the link was different.
If you have a better idea of how to simplify it I'm open to ideas, but I don't think `uname` is enough.